### ISSUE TITLE
v1.1.1 crasht Atmos 0.12.0

### ISSUE THREAD

**Issue by Conan179** [Sun-Research-University/sys-clk-Overlay#28]
Hello@all
I have a problem.
On my Switch FW 10.0.2 with atmos, atmos crasht complet, whe i try to open sys-clk-Overlay v1.1.1.
My systemmodule are only, sys-clk and Tesla.
[01590925677_420000000007e51a.log](https://github.com/Sun-Research-University/sys-clk-Overlay/files/4707604/01590925677_420000000007e51a.log)
With v1.1, i dont have problems.

**Comment by gabehxd** [comment]
can you share you config for sys clk

**Comment by PyroXFire** [comment]
I also am running 10.0.2 and have the same issue (renamed config from ini to txt because github didn't like it being an ini)
now launching a game I have a config setup for works. (ie i have ion fury setup via the config editor nro and it seems to work fine) but the second i open the overlay instant crash.


[01590952761_420000000007e51a.log](https://github.com/Sun-Research-University/sys-clk-Overlay/files/4708334/01590952761_420000000007e51a.log)

[config.txt](https://github.com/Sun-Research-University/sys-clk-Overlay/files/4708331/config.txt)

**Comment by PyroXFire** [comment]
also just tried and like the other user v1.1 works just fine for me

**Comment by Conan179** [comment]
In the /config/sys-clk/config.ini
are only this:
[values]
; Defines how often sys-clk log temperatures, in milliseconds (set 0 to disable)
temp_log_interval_ms=0
; Defines how often sys-clk writes to the CSV, in milliseconds (set 0 to disable)
csv_write_interval_ms=0

; Example #1: BOTW
; Overclock CPU when docked
; Overclock MEM to docked clocks when handheld
;[01007EF00011E000]
;docked_cpu=1224
;handheld_mem=1600

; Example #2: Picross
; Underclock to save battery
;[0100BA0003EEA000]
;handheld_cpu=816
;handheld_gpu=153
;handheld_mem=800

**Comment by gabehxd** [comment]
We found the source of the issue and we're trying to find fixes

**Comment by ghost** [comment]
Can this be solved by rolling back an update and if so, would that cause any potential issues?

**Comment by PyroXFire** [comment]
> Can this be solved by rolling back an update and if so, would that cause any potential issues?

yes this is what worked for me and been using it for a couple days no issues. So just roll back a version until the issue is fixed. Cheers 🍻

**Comment by gabehxd** [comment]
sorry for the wait, this should fix the issues: https://github.com/Sun-Research-University/sys-clk-Overlay/suites/753325834/artifacts/7760493

**Comment by ghost** [comment]
> sorry for the wait, this should fix the issues: https://github.com/Sun-Research-University/sys-clk-Overlay/suites/753325834/artifacts/7760493

Installed it and it's still causing crashes. Will be rolling back to v1.1 again.

**Comment by gabehxd** [comment]
can you provide a crash report

**Comment by gabehxd** [comment]
more details would also be appreciated when saying it crashed.

**Comment by gabehxd** [comment]
https://github.com/Sun-Research-University/sys-clk-Overlay/suites/753520155/artifacts/7764382 this should function properly

**Comment by ghost** [comment]
> sorry for the wait, this should fix the issues: https://github.com/Sun-Research-University/sys-clk-Overlay/suites/753325834/artifacts/7760493

Sorry for not being clearer! I installed the fix posted above and when launching sys-clk-Overlay via handheld mode, it crashes the Switch.
[01591244830_420000000007e51a.log](https://github.com/Sun-Research-University/sys-clk-Overlay/files/4727582/01591244830_420000000007e51a.log)

**Comment by ghost** [comment]
> https://github.com/Sun-Research-University/sys-clk-Overlay/suites/753520155/artifacts/7764382 this should function properly

Still crashes for me. Here's another log.
[01591246112_420000000007e51a.log](https://github.com/Sun-Research-University/sys-clk-Overlay/files/4727601/01591246112_420000000007e51a.log)

**Comment by Renari** [comment]
[01591240392_420000000007e51a.log](https://github.com/Sun-Research-University/sys-clk-Overlay/files/4733117/01591240392_420000000007e51a.log)

Same issue here.

**Comment by CaelThunderwing** [comment]
just got it here after updating,  didnt matter if i had no config blank or pre-existing from using sys-clk editor. 
[01591318121_420000000007e51a.log](https://github.com/Sun-Research-University/sys-clk-Overlay/files/4733316/01591318121_420000000007e51a.log)

**Comment by gabehxd** [comment]
Please try this fix: https://github.com/Sun-Research-University/sys-clk-Overlay/suites/761957801/artifacts/7918844 (Haven't had time to test myself)

**Comment by Conan179** [comment]
Nope, crashed again.
[01591397420_420000000007e51a.log](https://github.com/Sun-Research-University/sys-clk-Overlay/files/4738810/01591397420_420000000007e51a.log)

**Comment by MadBeastMSE** [comment]
> Please try this fix: https://github.com/Sun-Research-University/sys-clk-Overlay/suites/761957801/artifacts/7918844 (Haven't had time to test myself)

it crashes still even the older version crashes for me

**Comment by gabehxd** [comment]
@MadBeastMSE if v1.1 crashed i need a crash report

**Comment by MadBeastMSE** [comment]
@SunTheCourier 
https://gofile.io/d/aPXKcJ
this is my crash report

**Comment by Avrution** [comment]
Not sure if it is the same thing, but as soon as I click on the overlay option my switch reboots (no actual crash message)

**Comment by EXO5KELETON** [comment]
I can confirm that v1.1 works on a clean vanilla install on AMS 12 with Horizon firmware 10.0.4

**Comment by Avrution** [comment]
So, I had updated through the HB store and that caused the crash/reboot - updated directly from the files here and everything works, but it only shows as version 1.0 even with the latest file

**Comment by EXO5KELETON** [comment]
How did you manage that with no ovl for 1.1.1?
